scripts/provisioning/setupcmd.py

```diff
@@ -41,6 +42,8 @@ class SetupCmd(object):
   hosts_list = None
   s3_prov_config = "/opt/seagate/cortx/s3/mini-prov/s3_prov_config.yaml"
   _preqs_conf_file = "/opt/seagate/cortx/s3/mini-prov/s3setup_prereqs.json"
+  services_list = ["haproxy","s3backgroundconsumer","slapd","s3authserver"]
+  all_services_instances_list = ["s3backgroundproducer@*", "s3server@*"]
```

Comment:
Use  's3backgroundproducer' and move to services_list

Reply:
done